src/website/src/app/documentation/demos/forms/forms.demo.html

```diff
@@ -166,11 +166,20 @@ <h2>Forms Accessibility</h2>
                 find example markup for making the forms and form controls accessible.
             </p>
 
-            <p>
-                Finally, for screen readers, if there are any controls with errors after the submit action an
-                <code class="clr-code">aria-live</code> region will be updated with the associated labels for each
-                control in the error state.
-            </p>
+            <div class="alert alert-warning">
+                <div class="alert-items">
+                    <div class="alert-item static">
+                        <div class="alert-icon-wrapper">
+                            <clr-icon class="alert-icon" shape="exclamation-triangle"></clr-icon>
+                        </div>
+                        <span class="alert-text">
+                            As from V3 we no longer provide <code class="clr-code">aria-live</code> support as part of the component.
```

Comment:
Since v3, we no longer handle setting `aria-live` and announcing the message for you by default. Based on the application use case, you can use the new `ClrAriaLiveService` to make announcements when they make sense for a user to hear about updates or loading status changes.